lstchain/scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -145,8 +149,10 @@ def main():
             cmd = f"lstchain_create_calibration_file " \
                   f"--input_file={input_file} --output_file={output_file} "\
                   f"--EventSource.max_events={max_events} " \
-                  f"--LSTEventSource.EventTimeCalculator.dragon_reference_time={int(run_info['ucts_t0_dragon'])} " \
-                  f"--LSTEventSource.EventTimeCalculator.dragon_reference_counter={int(run_info['dragon_counter0'])} " \
+                  f"--EventSource.default_trigger_type=tib " \
```

Comment:
is this expected to change at some point or run always using tib?

Reply:
Hi @morcuended,
what do you mean with "at some point"? 
For old files (including run 2006 in the test data), ucts give wrong pedestal trigger values.
In general for the calibration data tib seems more reliable (it is what I used in the script before), but I can not exclude that for some runs ucts is better. Difficult to choose, do you have a strong opinion on that?

Reply:
Not really, see comment above. I was wondering if the event source could choose which one is better. But OK for me leaving it as 'tib'